content/en/docs/architecture/images.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "OCP Builder Images"
+date: 2020-01-22T23:27:39-04:00
+draft: false
+---
+
+Many components of OpenShift Container Platform 4 are tested and released as a container images (RPMs are outside the scope of this document). When a pull request for a component repository is created, a CI job builds any container images for the component using the code in the PR. We call these images _CI images_. When OCP is released as a product, component container images are built by the ART team using OSBS. We call these images _productized images_. In order to ensure that tests for pull requests give high fidelity signal for the final product, we ensure that the build process of the _CI images_ closely mirrors that of the _productized images_. This document overviews the build processes of _CI_ and _productized_ images.
```

Comment:
```suggestion
Many components of OpenShift Container Platform 4 are tested and released as a container images. When a pull request for a component repository is created, a CI job builds any container images for the component using the code in the PR. We call these images _CI images_. When OCP is released as a product, component container images are built by the ART team using OSBS. We call these images _productized images_. In order to ensure that tests for pull requests give high fidelity signal for the final product, we ensure that the build process of the _CI images_ closely mirrors that of the _productized images_. This document overviews the build processes of _CI_ and _productized_ images.
```

Reply:
The rest of the paragraph is very clear in what we will talk about in this doc - no need to talk about what we will *not* talk about